src/backend/communities.nim

```diff
@@ -92,7 +91,7 @@ proc createCommunity*(
       "historyArchiveSupportEnabled": historyArchiveSupportEnabled,
       "pinMessageAllMembersEnabled": pinMessageAllMembersEnabled,
       "banner": bannerImage,
-      "encrypted": encrypted
+      "encrypted": false
```

Comment:
Maybe we can add a comment like
```nim
# Deprecated: We send false for new communities because it gets managed on status-go's side when adding or removing permissions
```